tests/run_test.sh

```diff
@@ -80,9 +80,19 @@ cp ${PATHRT}/modules.fv3_${COMPILE_NR}             modules.fv3
 # Get the shell file that loads the "module" command and purges modules:
 cp ${PATHRT}/../NEMS/src/conf/module-setup.sh.inc  module-setup.sh
 if [[ $FV3 = 'true' ]]; then
-  cp ${PATHRT}/parm/post_itag itag
-  cp ${PATHRT}/parm/postxconfig-NT.txt postxconfig-NT.txt
-  cp ${PATHRT}/parm/postxconfig-NT_FH00.txt postxconfig-NT_FH00.txt
+  if [[ $POSTAPP = 'global' ]]; then
```

Comment:
This runs counter the idea that the regression testing scripts are independent of the tests. Shouldn't this go into the relevant files in `tests/fv3_conf/` ?

Reply:
Please see the comments above. By default, all the global/regional tests in current RT have those files present. This requires a separate fix.

Reply:
Why can't we fix this as part of this PR? If there is a bunch of tests that need those files, then either add it to the relevant files in `tests/fv3_conf` or create a little helper script that lives in `tests/fv3_conf` and gets called from the relevant run scripts in `tests/fv3_conf`? If it's about not having to rerun the regression test, then this should be fixed as soon as possible (and an issue needs to be created).